internal/controllers/certmanager.go

```diff
@@ -122,6 +122,35 @@ func (r *Reconciler) setupTLS(ctx context.Context, cr *model.CryostatInstance) (
 		return nil, err
 	}
 
+	// Copy Cryostat CA secret in each target namespace
+	for _, ns := range cr.TargetNamespaces {
+		secret, err := r.GetCertificateSecret(ctx, caCert)
+		if err != nil {
+			return nil, err
+		}
+		namespaceSecret := secret.DeepCopy()
+		namespaceSecret.Namespace = ns
+		err = r.createOrUpdateSecret(ctx, namespaceSecret, cr.Object, func() error {
+			if secret.StringData == nil {
+				secret.StringData = map[string]string{}
+			}
```

Comment:
I think this should be `namespaceSecret` inside the closure since that's what we're intending to modify.

We should copy the `Data` from the original secret here too. Just `namespaceSecret.Data = secret.Data` should work. Also we should change this initialization to `Data` instead of `StringData` because of this. The latter is a write-only convenience field that the server Base64 encodes and moves to `Data`.